e2e-tests/cypress/e2e/journeys/insurance/your-business/company-details/company-details-submit.spec.js

```diff
@@ -12,13 +12,14 @@ const {
     },
     YOUR_COMPANY: {
       TRADING_NAME,
+      TRADING_ADDRESS,
     },
   },
 } = FIELD_IDS.INSURANCE;
 
 const COMPANY_DETAILS_ERRORS = ERROR_MESSAGES.INSURANCE.EXPORTER_BUSINESS;
 
-describe("Your business - company house search - As an Exporter I want to enter details about my business in 'your business' section", () => {
+describe("Your business - As an Exporter I want to enter details about my business in 'your business' section", () => {
```

Comment:
Can we prefix with Insurance, the section and page? Great for debugging

`context('Insurance - Policy and exports - Type of policy page - As an exporter, ...`

Reply:
done